crates/api_models/src/payments.rs

```diff
@@ -285,6 +285,10 @@ pub struct PaymentsRequest {
     /// Any user defined fields can be passed here.
     #[schema(value_type = Option<Object>, example = r#"{ "udf1": "some-value", "udf2": "some-value" }"#)]
     pub udf: Option<pii::SecretSerdeValue>,
+
+    /// If enabled denotes that the payment is requeued
+    #[serde(default)]
+    pub requeue: bool,
```

Comment:
I think a confirm call can do both reque and retry. As far as I understand, the retry field has affect after the connector is called and if it fails with some error that can be retried, then it will be retried. But in case of requeue, it has affect even before the connector is called.